src/commonMain/kotlin/com/github/quillraven/fleks/collection/ListMutableEntityBag.kt

```diff
@@ -0,0 +1,653 @@
+@file:Suppress("OVERRIDE_BY_INLINE")
+
+package com.github.quillraven.fleks.collection
+
+import com.github.quillraven.fleks.Entity
+import kotlin.math.max
+import kotlin.math.min
+
+class ListMutableEntityBag(
+    @PublishedApi
+    internal val list: MutableList<Entity>
```

Comment:
Please rename it to `values` to be consistent with the other bag implementation.